config/routes.rb

```diff
@@ -0,0 +1,31 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root 'works#homepage'
+
+  resources :votes, only: [:index, :show, :create]
+
+  resources :works do
+    resources :votes, only: [:index, :create, :update]
+    member do
+      post 'upvote'
+    end
+  end
+
+  resources :works do
+
+   end
```

Comment:
When you have `resources :works` above this block already declared, you don't need to re-declare it empty down here. The above block accomplishes the whole "making routes for `works" bit!